css/styles.css

```diff
@@ -0,0 +1,266 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+    float: center center;
+    background-color: lightgrey;
+    font-family: Merriweather;
+  }
+
+img.background {
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 100%;
+    float: center;
+  }
+
+header  {
+  color: white;
+  background-color: #000000;
+  height: 100px;
+  border: 1px;
+  margin: 0px;
+}
+
+a {
+  color: white;
+}
+
+a:hover {
+  color: #6A5ACD;
+}
+
+h1  {
+  color: yellow;
+  padding-left: 30px;
+  padding-bottom: 5px;
+  font-size: 35px;
+  margin: 1px 8px 1px 8px;
+}
+
+h2  {
+  height: 50px;
+  color: #000000;
+  text-align: center;
+  margin: 5px;
+}
+
+.nav-items {
+  color: white;
+  width: 1020px;
+  padding: 1px 10px 1px 10px;
```

Comment:
@Schultisal this could be written as `padding: 1px 10px`.  Where the first value affects the top and bottom and the second value affects left and right.